src/app/models/profile.ts

```diff
@@ -6,8 +6,11 @@ export class Profile {
     bio?: string;
     username?: string;
     url?: string;
-    emails?: Email[];
-    primaryEmail?: Email;
+    emails?: string[];
```

Comment:
yes, then you can have as many new types as you want and you fetch them from the db. We can take another level and create email types so that we are sure they will match.